### DISCUSSION TITLE
A walk-through tutorial on using the Create Block Theme plugin

### DISCUSSION THREAD

**Discussion by tinjure20** [WordPress/developer-blog-content#171] [Topic Ideas]
I am a huge fan of the [Create Block Theme](https://wordpress.org/plugins/create-block-theme/) (CBT) plugin, which is a must-have in my workflows, serving as a swish knife like tool for creating block themes. I had made a draft document about how to use the plugin a while ago, but it may need to be updated with the latest WP & CBT plugin releases.

IMO, this would be a useful walk-through tutorial topic, and adding it to the list for discussion with a link to the Google Doc.

https://docs.google.com/document/d/1uh4Pn-jtVXJ1pmbcCGs7bYAhPFs12L-qVqdxIhu1qww/edit?usp=sharing

**Comment by bph** [comment]
Please continue conversation on the issue

- https://github.com/WordPress/developer-blog-content/issues/173

This discussion space is locked in favor of the issue.

**Comment by bph** [comment]
Published: [A walk-through tutorial on using Create Block Theme plugin](https://developer.wordpress.org/news/2023/12/05/a-walk-through-tutorial-on-using-create-block-theme-plugin/)
